src/media/ros/ros_writer.cpp

```diff
@@ -207,8 +208,7 @@ namespace librealsense
         imu_msg.header.seq = static_cast<uint32_t>(frame.frame->get_frame_number());
         std::chrono::duration<double, std::milli> timestamp_ms(frame.frame->get_frame_timestamp());
         imu_msg.header.stamp = rs2rosinternal::Time(std::chrono::duration<double>(timestamp_ms).count());
-        std::string TODO_CORRECT_ME = "0";
-        imu_msg.header.frame_id = TODO_CORRECT_ME;
+        imu_msg.header.version = "1"; // used to distinguish between old rosbag and new rosbag that contains depth units in frame metadata
```

Comment:
Add comment that the field is unused and therefore assigned for ROSbag versions control